modules/mobile/app/services/mobile/v0/lighthouse_claims/claims_index_interface.rb

```diff
@@ -10,96 +10,63 @@ module LighthouseClaims
       class ClaimsIndexInterface
```

Comment:
All of these changes are just refactoring to simplify the code. I started refactoring before I realized that my changes needed to take place in the claims overview errors file. But I think these changes simplify the code and make it easier to understand.